packages/platform-android/native_modules.gradle

```diff
@@ -326,22 +469,40 @@ ext.applyNativeModulesAppBuildGradle = { Project project, String root = null ->
 
   def generatedSrcDir = new File(buildDir, "generated/rncli/src/main/java")
   def generatedCodeDir = new File(generatedSrcDir, generatedFilePackage.replace('.', '/'))
+  def generatedJniDir = new File(buildDir, "generated/rncli/src/main/jni")
 
   task generatePackageList {
     doLast {
       autoModules.generatePackagesFile(generatedCodeDir, generatedFileName, generatedFileContentsTemplate)
     }
   }
 
+  task generateNewArchitectureFiles {
+    doLast {
```

Comment:
I would refrain from using `doLast`. Those are terrible from the caching point of view (as they invalidate the gradle caches). We should move this to a proper task.

Not a blocker though :), just a heads up

Reply:
Sure! I'm not a heavy user of Gradle, so I'd appreciate any contributions in code or pointers on what to leverage exactly :)